edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -69,14 +69,27 @@ const LectureScreen: ApplicationRoute = () => {
         }
     }, [lectureDoc]);
 
+    useFocusEffect(
+        useCallback(() => {
+            // This effect runs every time the screen is unfocused or focused
+            return () => {
+                // Unlock orientation whenever the screen loses focus (navigating back)
+                ScreenOrientation.unlockAsync();
+            };
+        }, [])
+    );
 
     // Landscape display for the screen
     const setLandscape = async () => {
         await ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.LANDSCAPE);
     };
+    // Landscape display for the screen
+    const setPortrait = async () => {
+        await ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.PORTRAIT);
+    };
 
     useEffect(() => {
-        setLandscape();
+        setPortrait();
```

Comment:
It better this way. It comes the more naturally for the user. Great job.

Reply:
thank you